src/MoBi.Presentation/Presenter/ChartPresenter.cs

```diff
@@ -23,6 +23,7 @@
 using OSPSuite.Utility.Collections;
 using OSPSuite.Utility.Events;
 using OSPSuite.Utility.Extensions;
+using static OSPSuite.Assets.Captions;
```

Comment:
why this?
(wondering because there are no other code changes in the file)

Reply:
true, this slipped my attention - just something that was left there. Removed.